actioncable/lib/action_cable/engine.rb

```diff
@@ -2,11 +2,13 @@
 require "action_cable"
 require "action_cable/helpers/action_cable_helper"
 require "active_support/core_ext/hash/indifferent_access"
+require "action_dispatch/routing/inspector"
+require "action_cable/route_wrapper"
 
 module ActionCable
   class Railtie < Rails::Engine # :nodoc:
     config.action_cable = ActiveSupport::OrderedOptions.new
-    config.action_cable.url = '/cable'
+    config.action_cable.mount_path = '/cable'
```

Comment:
Not sure I understand this. We still need to allow someone to set the URL even if they're not mounting in-app. So need to retain the #url and then in addition add #mounted = true/false to decide whether to mount in-app. //cc @matthewd